extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -557,12 +562,16 @@ export class AmpSlideScroll extends BaseSlides {
       // Set the correct direction.
       direction = direction < 0 ? 1 : -1;
     }
-    this.analyticsEvent_('amp-carousel-change');
+    const vars = {
+      'fromSlide': this.dataSlideIdArr[this.slideIndex_],
+      'toSlide': this.dataSlideIdArr[newSlideIndex],
+    };
```

Comment:
@avimehta @rudygalfi @cramforce @dvoytenko - does this sound reasonable as something that you would like to have on carousel analytics events?